templates/zerver/integrations/development/dev_panel.html

```diff
@@ -7,7 +7,7 @@
 {% endblock %}
 
 
-{% block content %}
+{% block portico_content %}
```

Comment:
Can you talk about why you made this change, and/or move it to a separate commit with its own commit message?

Reply:
I consider dev_panel to be a portico page, but it seems to me that it's a typo here for it to not use portico_content. If I don't make the change, the alert-box won't appear on this page and the popup will not work.

Reply:
Makes sense -- I think there may have been some intention behind that decision.  

Aha!  With `git log -p`, I found 1a12e112d9add36add23a3fdaacf33636dc8bb9b, which did intend to convert this into a portico page, so I think this change is correct.